### PR TITLE
VPN-5594 Use a diffrent Settings File for testing and running the client!

### PR DESCRIPTION
**Pull request by strseb**
## Description

The Functional tests are using the same Settings file as the normal vpn client. 
This set's settings like "only use localhost requests" which means i always need to nuke that file after running tests. 
Let's have this file per-test suite.

### CODE REVIEW

**Review by brizental — APPROVED** [review]
!!! GO BASTI !!! 💯 🏆

**Comment by brizental** [comment]
There is a ticket for this https://mozilla-hub.atlassian.net/browse/VPN-5594, btw.

File: src/settings/settingsmanager.cpp [hunk 1]
```diff
@@ -27,6 +26,15 @@ constexpr const char* SETTINGS_ORGANIZATION_NAME = "mozilla";
 constexpr const char* SETTINGS_ORGANIZATION_NAME = "mozilla_testing";
 #endif
```
Comment (brizental) [inline src/settings/settingsmanager.cpp:27]: I thin this doesn't need to be different anymore now that the app name is different.

**Review by mcleinman — APPROVED** [review]
Thank you!!!

**Pull request merged**